playstore/notification.go

```diff
@@ -48,6 +49,19 @@ type DeveloperNotification struct {
 	TestNotification           TestNotification           `json:"testNotification,omitempty"`
 }
 
+// DeveloperNotificationNew is sent by a Pub/Sub topic.
+// Detailed description is following.
+// https://developer.android.com/google/play/billing/rtdn-reference#json_specification
+type DeveloperNotificationNew struct {
```

Comment:
Can it be `DeveloperNotificationV2` as `New` may cause confusion at a later date? 🙏

Reply:
> Can it be `DeveloperNotificationV2` as `New` may cause confusion at a later date? 🙏

ok